src/database/migrations/1602107443079-AddConfirmationToUsers.ts

```diff
@@ -0,0 +1,36 @@
+import { MigrationInterface, QueryRunner, TableColumn } from 'typeorm';
+
+export class AddConfirmationToUsers1602107443079 implements MigrationInterface {
```

Comment:
I have a question here, given the base project is supposed to be the starting point for the db, is it useful to keep some migrations like this one?

Reply:
This is outdated. We are using migrations now